### ISSUE TITLE
getting output image between 0 to 100 in pix2pix

### ISSUE THREAD

**Issue by sepideh-srj** [junyanz/pytorch-CycleGAN-and-pix2pix#1146]
I want to have the output image between 0 to 100 instead of 0 to 1 which is what the pix2pix network will generate.
I tried to add the following parameter :
self_fake_B_new = self.fake_B * 100
and changed the loss functions in forward and backward from fake_B to fake_B_new, but it's not working. 
Any idea what I'm doing wrong?

**Comment by JohanRaniseth** [comment]
My guess is that the code is dependent on having all values between 0 and 1. Maybe it screws with the loss values, weight updates, or something else.

If the original code works for you it's probably safer to just scale up the images to 0 to 100 after the output image is produced, i.e through using a .py file on an output image collection, or find the code where output image is written to file and scale up there.

**Comment by sepideh-srj** [comment]
> My guess is that the code is dependent on having all values between 0 and 1. Maybe it screws with the loss values, weight updates, or something else.
> 
> If the original code works for you it's probably safer to just scale up the images to 0 to 100 after the output image is produced, i.e through using a .py file on an output image collection, or find the code where output image is written to file and scale up there.

Thank you Johan, I will try that.
